### PR TITLE
Feature/add new book

### PR DESCRIPTION
**Pull request by Mayurp375**
i have did create book data. can you please check and merge the code

### CODE REVIEW

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    Date publishedDate;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
```
Comment (ajinkyabridgelabz) [inline src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java]: Please add the lombok support and remove the getters and setters.

File: entitydetails.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+"id": 1,
+"title":"{{$randomProductName}}",
+"author":"{{$randomFirstName}}",
```
Comment (ajinkyabridgelabz) [inline entitydetails.md]: Don't use the postman variables

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/service/BookService.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.bridgelabz.onlinelibrarymanagementsystem.service;
+
+import com.bridgelabz.onlinelibrarymanagementsystem.entity.Book;
+
+public interface BookService {
```
Comment (ajinkyabridgelabz) [inline src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/service/BookService.java:5]: Please don't send a string as a response.

File: src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    public void setPublishedDate(Date publishedDate) {
+        this.publishedDate = publishedDate;
+    }
+
+    public Book() {
+    }
+
+    public Book(int id, String title, String author, String genre, String isbn, Date publishedDate) {
+        this.id = id;
+        this.title = title;
```
Comment (ajinkyabridgelabz) [inline src/main/java/com/bridgelabz/onlinelibrarymanagementsystem/entity/Book.java]: For constructor give lombok supprot

**Comment by ajinkyabridgelabz** [comment]
Please sort all the issues and resend me the request to merge.

**Comment by Mayurp375** [comment]
issue is resolved kindly look up and merge it

**Review by ajinkyabridgelabz — APPROVED** [review]
Ok fine all the changes are done. I will merge now.

**Review by ajinkyabridgelabz — APPROVED** [review]

**Pull request merged**